### DISCUSSION TITLE
Understanding the nomenclature of log files in _diag/pages folder of self-hosted runners?

### DISCUSSION THREAD

**Discussion by kartikdc** [community/community#27051] [Actions]
<p>Hello. I plan on collecting the logs in the <code>_diag/pages</code> folder on self-hosted runners. But I can’t make sense of the file names. The filenames are something like this: <code>d5b17915-27a2-4695-abff-713e234e6908_ebc7c1bb-fed3-5b92-08ac-8c96008a69ca_1.log</code>.</p>
<p>Does anyone know these mean? Thanks a lot.</p>

**Comment by joshmgross** [comment, marked as answer]
<p>:wave: Hey <a class="mention" href="/kartikdc">@kartikdc</a>,</p>
<p>These file names contain the internal IDs we use to differentiate different runs, jobs, and steps.</p>

**Comment by kartikdc** [comment]
<p>Thanks very much buddy</p>

**Comment by CNTKR** [comment]
Hi, I have come across this 'pages' folder in my self-hosted agent and am not able to figure out why it is being used and in which scenarios logs get generated in this folder. I would really appreciate it if anyone has an idea about it and can explain it in detail. Thank you in advance!
